app/views/users/show.html.erb

```diff
@@ -12,7 +12,7 @@
   <strong>Bio:</strong>
   <%= @user.bio %>
 </p>
-<div id="avatar"><%= image_tag(@user.avatar_url) if @user.avatar? %></div>
+<div id="avatar"><%= image_tag(@user.avatar_url(:thumb)) if @user.avatar? %></div>
```

Comment:
is `avatar_url` an `rmagick` helper method?  i couldn't find it defined anywhere.  i'm guessing it is  :)